samples/snippets/csharp/VS_Snippets_CLR_System/system.DateTime.Parse/cs/Parse6.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+
+public class Example
+{
+   public static void Main()
+   {
+      ValueTuple<string,string>[] dateInfo = { ("08/18/2018 07:22:16", "String with a date and time component"),                             
```

Comment:
I'd write the declaration like this:

```csharp
(string dateAsString, string Description)[] dateInfo = {}
```

Then, you can use `dateAsString` and `Description` instead of `Item1` and `Item2`